voluptuous_openapi/__init__.py

```diff
@@ -108,15 +109,41 @@ def ensure_default(value: dict[str:Any]):
             pval = ensure_default(pval)
 
             if isinstance(pkey, vol.Any):
-                for val in pkey.validators:
-                    if isinstance(val, vol.Marker):
-                        if val.description:
-                            properties[str(val.schema)] = pval.copy()
-                            properties[str(val.schema)]["description"] = val.description
+                # Handle Required(Any(...)) pattern for anyOf constraints
+                if isinstance(key, vol.Required):
+                    # Extract candidate keys from Any validator
+                    candidate_keys = []
+                    for val_item in pkey.validators:
+                        if isinstance(val_item, vol.Marker):
+                            candidate_keys.append(str(val_item.schema))
                         else:
-                            properties[str(val)] = pval
+                            candidate_keys.append(str(val_item))
+                    
+                    # Check if the value is object (wildcard for presence-only validation)
+                    if value is object:
```

Comment:
What are examples that cause this to be true? First time i have seen this syntax and testing it locally in a pyhon shell i'm not seeing how to make this return true.